src/docs/deploy/regular.rst

```diff
@@ -199,11 +215,11 @@ is going to be used next time we do deployment.
 
     git clone git@github.com:mozilla/release-services.git
     cd release-services
-    git co -b version-bump origin/master
+    git checkout -b version-bump origin/master
     echo "$((($(cat VERSION)) + 1))" | tee VERSION2
     sed -i -e "s|base-$(cat VERSION)|base-$(cat VERSION2)|" .taskcluster.yml
     mv VERSION2 VERSION
-    git commit -a -m "Deploying v$(cat VERSION) (and bumping version to v$((($(cat VERSION)) + 1)))"
+    git commit -a -m "Deploying v$((($(cat VERSION)) - 1)) (and bumping version to v$(cat VERSION))"
```

Comment:
Sure about this? I always get confused on releases about the version number...

Reply:
Pretty sure, i just did 0dbe6e7ca79276e5c0ee6846e0435da7aba76dc8 (for the next release) with this.
The previous line would work *before* bumping `VERSION`